content/docs/examples/advanced-gateways/http-proxy/index.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Connect to an external HTTPS proxy
+description: Describes how to configure Istio to let applications use an external HTTPS proxy.
+weight: 60
+keywords: [traffic-management,egress]
+---
+The [Configure an Egress Gateway](/docs/examples/advanced-gateways/egress-gateway/) showed how you can direct traffic to
+external services from your mesh via an Istio edge component called _Egress Gateway_. However, there are cases when you
+must use an external, legacy (non-Istio) HTTPS proxy to access external services. For example, your company may already
+have such a proxy in place and all the applications within the organization may be required to direct their traffic
+through it. This example shows how to enable access to an external HTTPS proxy.
+
+## Before you begin
+
+*   Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
+
+*   Start the [sleep]({{< github_tree >}}/samples/sleep) sample
+    which will be used as a test source for external calls via the proxy.
+
+    If you have enabled
+    [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection), do
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/sleep/sleep.yaml@
+    {{< /text >}}
+
+    otherwise, you have to manually inject the sidecar before deploying the `sleep` application:
+
+    {{< text bash >}}
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@)
+    {{< /text >}}
+
+    Note that any pod that you can `exec` and `curl` from would do.
+
+*   Create a shell variable to hold the name of the source pod for sending requests to external services.
+    If you used the [sleep]({{<github_tree>}}/samples/sleep) sample, run:
+
+    {{< text bash >}}
+    $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
+    {{< /text >}}
+
+## Deploy an HTTPS proxy
+
+For this example, to simulate a legacy proxy, you deploy an HTTPS proxy inside your cluster. Also, to simulate a more
+realistic proxy that is running outside of your cluster, you will address the pod of the proxy by its IP address and
+not by a Kubernetes service.
+You can use any HTTPS proxy that supports HTTP Connect. We used [Squid](http://www.squid-cache.org).
```

Comment:
Emphasize and focus on the current flow. By switching the order of the sentences, you achieve that emphasis. Remember to avoid past tense.
```suggestion
This example uses [Squid](http://www.squid-cache.org) but you can use any HTTPS proxy that supports HTTP Connect.
```

Reply:
> We used Squid

The issue is a bit complicated. If I say "we use Squid", in the present tense, it would sound like we and not the reader use Squid in this example. The meaning here is "We decided to use Squid to describe the example, so you, the reader, will use the Squid in this example".

Reply:
Applied in 4954637b98186dbb4b447c448651d4fad7b90e3e